### PR TITLE
org.desmume.DeSmuME_0.9.13

### PR DESCRIPTION
**Pull request by Zer077**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

### CODE REVIEW

File: org.desmume.DeSmuME_0.9.13.json [hunk 1]
```diff
@@ -0,0 +1,62 @@
+{
+    "app-id": "org.desmume.DeSmuME_0.9.13",
```
Comment (hfiguiere) [inline org.desmume.DeSmuME_0.9.13.json:2]: version number are NEVER in the app-id

File: org.desmume.DeSmuME_0.9.13.json [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "21.08",
```
Comment (hfiguiere) [inline org.desmume.DeSmuME_0.9.13.json:4]: this runtime is obsolete. It will be EOL next month.

**Comment by hfiguiere** [comment]
also desmume is already on flathub

https://flathub.org/apps/org.desmume.DeSmuME

**Comment by Zer077** [comment]
> also desmume is already on flathub
> 
> https://flathub.org/apps/org.desmume.DeSmuME

but it is not updated, I can not make a flatpak of the application to be able to keep it updated?

**Comment by hfiguiere** [comment]
that's not how it works.

**Comment by Zer077** [comment]
> that's not how it works.

And how does it work then? since the owner of the manifest (which is the file to be updated) does not accept pull requests

**Comment by hfiguiere** [comment]
which PR was rejected?

**Comment by Zer077** [comment]
> which PR was rejected?

https://github.com/flathub/org.desmume.DeSmuME/pull/10 this pull request is 1 year old and never was meged, the owner does not make the merger

**Comment by Zer077** [comment]
> that's not a refusal.

but if you don't like it, it's useless, right? or can I update it without the owner's consent?

**Comment by hfiguiere** [comment]
we don't take new version as new submissions.

**Comment by Zer077** [comment]
> Contributor

So are there options for completely abandoned applications like this? asked

**Pull request closed without merging**